compiler/qsc/src/lib.rs

```diff
@@ -4,13 +4,57 @@
 #![warn(clippy::mod_module_files, clippy::pedantic, clippy::unwrap_used)]
 #![allow(clippy::missing_errors_doc, clippy::missing_panics_doc)]
 
+use std::str::FromStr;
+
 pub mod compile;
 pub mod error;
 pub mod incremental;
 pub mod interpret;
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum TargetProfile {
```

Comment:
Can we pull this code into a module? Seems out of place sitting in the top level `lib` when everything else here is a reexport.

Reply:
Agreed, do you have any suggestions?